src/content/1.7/development/page-reference/back-office/order/refunds/_index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Refunds
+menuTitle: Refunds
+---
+
+# Cancellations and refunds
+{{< minver v="1.7.7" title="true" >}}
+
+{{% notice info %}}
+For a full specification of how these features work, you can read [the functional specifications](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/orders/orders/Order%20Page%20View.md#merchandise-return-has-to-be-enabled-if-the-merchant-want-to-use-the-standard-refund-partial-refund-and-return-product-feature)
+{{% /notice %}}
+
+## Order history and corresponding cancellation types
+
+{{% notice info %}}
+Refund features are not activated by default. To activate them, go to the "Merchandise returns" page under the "Customer Service" menu, and activate the "Product returns" option at the bottom of the page. This will apply to all products and orders.
+{{% /notice %}}
+
+In the order detail page, **return product**, **cancel product**, and **standard refund** buttons can only exist exclusively, following these rules:
+
+- If the order has been shipped, then the available action will be **return product**.
+
+- If the order has been paid but not shipped yet, then the available action will be **standard refund**.
+
+- In all other cases, the available action is **cancel product**.
+
+The **Partial refund** button is available once a payment was made on the order, it can exist jointly with other refunds, and it exists even if the "Product returns" option was not enabled.
+
+
+## Performing cancellation actions programmatically
+
+### Introduction
+
+Since prestashop 1.7.7 cancellation actions (that is to say **cancel product**, **return product**, **standard refund** and **partial refund**) are implemented following the CQRS design pattern.
```

Comment:
```suggestion
Since Prestashop 1.7.7 cancellation actions (that is to say **cancel product**, **return product**, **standard refund** and **partial refund**) are implemented following the CQRS design pattern.
```